libs/compute_api/src/requests.rs

```diff
@@ -12,3 +12,11 @@ use serde::Deserialize;
 pub struct ConfigurationRequest {
     pub spec: ComputeSpec,
 }
+
+#[derive(Deserialize, Debug)]
+pub struct SetRoleGrantsRequest {
+    pub database: String,
```

Comment:
NIT: I wonder, does it make sense to use `PgIdent` here for database, schema and role?

Yes, it's currently defined as `pub type PgIdent = String;`, which as Conrad recently mentioned might not be idiomatic, but at least sends the right message that it should be escaped